src/servers/esl.js

```diff
@@ -13,15 +13,19 @@ class EslServer extends SocketServer {
   }
 
   holdCall(callId) {
+    this._getSocket(callId).event_json('CHANNEL_HANGUP');
+    this._getSocket(callId).filter('variable_conference_name', callId);
```

Comment:
Now, again ;) , why is the variable _conference_name_ here, while it is _conference_ everywhere else? 😉